src/components/Trends/TrendsPanel.less

```diff
@@ -313,6 +313,10 @@
 
 
   .layout-row {
+    &.disclaimer {
```

Comment:
You could consider making this more specific, like `chart-disclaimer`. `disclaimer` is slightly generic.